.gitignore

```diff
@@ -1,5 +1,6 @@
 .vscode
 .DS_Store
+gridRE_tests
```

Comment:
@pypapus I think if you add this to your global `.gitignore` file it won't show up here but will still ignore your test folder. Let me know if you need help with this.